dashboard/extract.py

```diff
@@ -0,0 +1,73 @@
+'''Contains get_connection function'''
+from os import environ as ENV
+import logging
+from dotenv import load_dotenv
+from psycopg2 import connect
+from requests import get
+from api_error import APIError
+
+URL = "https://api.open-meteo.com/v1/forecast"
+
+def get_connection():
+    '''Returns a psycopg2 connection'''
+    return connect(f"""dbname={ENV["DB_NAME"]} user={ENV["DB_USER"]}
```

Comment:
Do we need to load the dotenv here? Or could this function just be imported from another script?